src/main/java/bio/terra/common/logging/LoggingUtils.java

```diff
@@ -78,34 +77,30 @@ static JsonNode jsonFromString(String s) throws JsonProcessingException {
    * default Spring logging config (see resources/logback.xml) will be used.
    */
   protected static void initializeLogging(ConfigurableApplicationContext applicationContext) {
-    if (loggingInitialized) {
-      return;
-    }
-    loggingInitialized = true;
-
     ch.qos.logback.classic.Logger logbackLogger =
         (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
     ConfigurableEnvironment environment = applicationContext.getEnvironment();
 
     if (Arrays.stream(environment.getActiveProfiles()).anyMatch("human-readable-logging"::equals)) {
       System.out.println("Human-readable logging enabled, skipping Google JSON layout");
-      return;
-    }
+      // No action needed. The logback.xml file on the classpath contains a default human-readable
+      // logging layout.
+    } else {
+      GoogleJsonLayout layout = new GoogleJsonLayout(applicationContext);
+      layout.start();
 
-    GoogleJsonLayout layout = new GoogleJsonLayout(applicationContext);
-    layout.start();
+      LayoutWrappingEncoder encoder = new LayoutWrappingEncoder();
+      encoder.setLayout(layout);
+      encoder.start();
 
-    LayoutWrappingEncoder encoder = new LayoutWrappingEncoder();
-    encoder.setLayout(layout);
-    encoder.start();
+      ConsoleAppender appender = new ConsoleAppender();
+      appender.setName(TERRA_APPENDER_NAME);
+      appender.setEncoder(encoder);
+      appender.setContext(logbackLogger.getLoggerContext());
+      appender.start();
```

Comment:
🎵 the ApplicationContext on the layout and the layout on the encoder and the encoder on the appender and the appender on the logger and the green grass grows all around all around...